index.html.in

```diff
@@ -199,6 +199,9 @@
                                           <label class="pull-right">
                                               <input type="checkbox" id="instantTranslation" checked> <span data-text="Instant_Translation">Instant translation</span>
                                           </label>
+                                          <label class="pull-right">
+                                              <input type="checkbox" class="chainedTranslation" unchecked> <span data-text="Chained_Translation" class="chainedTranslation">Chained translation</span>
```

Comment:
I don't believe `unchecked` is a valid attribute.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement